.github/workflows/push.yml

```diff
@@ -22,6 +26,14 @@ jobs:
     - uses: actions/setup-python@v2
       with:
         python-version: '3.x'
+    - name: Cache tools
+      id: cache-linux
+      uses: actions/cache@v2
+      with:
+        path: |
+          ./tools/dist
+          /home/runner/arduino_ide
+        key: ${{ runner.os }}-${{ hashFiles('package/package_esp32_index.template.json', 'tools/get.py') }}
```

Comment:
Probably also need to hash `.github/scripts/install-arduino-ide.sh` since that determines the contents of `/home/runner/arduino_ide`